src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -92,20 +93,18 @@ def run(self, revision):
          * Run static analysis
          * Publish results
         '''
-        assert revision.mercurial is not None, \
-            'Cannot run without a mercurial revision'
 
         # Add log to find Taskcluster task in papertrail
         logger.info(
             'New static analysis',
             taskcluster_task=self.taskcluster_task_id,
             taskcluster_run=self.taskcluster_run_id,
             channel=settings.app_channel,
-            revision=revision,
+            revision=str(revision),
```

Comment:
Isn't the logger automatically converting to str?

Reply:
Apparently not, i got the python __repr__